pom.xml

```diff
@@ -95,7 +108,7 @@
         <sonar.projectKey>${github.organization}_${project.artifactId}</sonar.projectKey>
         <!--        <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>-->
         <sonar.java.checkstyle.reportPaths>target/checkstyle-result.xml</sonar.java.checkstyle.reportPaths>
-
+        <fabric8.version>4.10.3</fabric8.version>
```

Comment:
In 6.4 we will be using 5.3.x which is quite a significant upgrade. I do however understand this needs to work on 6.3.2

Reply:
Good to know, thanks.